ChangeLog

```diff
@@ -2,9 +2,19 @@ mbed TLS ChangeLog (Sorted per branch, date)
 
 = mbed TLS x.x.x branch released xxxx-xx-xx
 
+API Changes
+   * Extend the platform module with an abstraction mbedtls_platform_gmtime()
+     whose implementation should behave as a thread safe version of gmtime().
```

Comment:
Isn't "thread safe" an over statement given that we fall back to `gmtime` and the mutexes will not be enough for an application wide use. Complete thread safety may be out of scope of this solution. We can either enhance the documentation to explain the situation or not state to b thread safe.